web/src/main/java/com/bazaarvoice/emodb/web/scanner/control/LocalScanUploadMonitor.java

```diff
@@ -477,6 +477,12 @@ private void initializeAllActiveScans() {
     private void scheduleOverrunCheck(ScanStatus status) {
         final String scanId = status.getScanId();
 
+        // If temporal stash is disabled, this is likely not a traditional stash run, and it may take longer than 24 hours
```

Comment:
minor: I would prefer regular or daily word instead of traditional. traditional seems confusing as classic.

Reply:
agreed.